src/hooks/util/useNumpad.tsx

```diff
@@ -1,20 +1,20 @@
-import { Currency } from '@/types';
 import { useEffect, useState } from 'react';
 
 interface UseNumpadProps {
-   showDecimal?: boolean;
+   showDecimal: boolean;
 }
 
 export interface NumpadControls {
    numpadValue: string;
    numpadValueIsEmpty: boolean;
+   numpadAmount: number;
```

Comment:
I wonder if we should call the properties `numpadStringValue` and `numpadValue` to make it clearer what is the diff between the two